### PR TITLE
VAP-460 add nodule grid to biopsy & PET forms

### PR DESCRIPTION
**Pull request by djdenv**
This adds the nodule grid to Biopsy and PET forms. Some duplicate fields that were on the PET form such as nodule index were removed. TSV files updated.

### CODE REVIEW

File: docs/mockups/ctevaluation-elcap.html [hunk 1]
```diff
@@ -5368,158 +5392,568 @@ <h4>
...
+                    $("[name='ceanod']").val(noduleCount === 0 ? "n" : "y");
+                }
+            });
 
-        $(".add-nodule").on('click', $noduleGrid.addNodule);
-        $(".remove-nodule").on('click', $noduleGrid.removeNodule);
-        exclusiveCheckbox("[name='cectin']", "tr" /* the common parent of all checkboxes of this name*/);
+            $(".add-nodule").on('click', $noduleGrid.addNodule);
+            $(".remove-nodule").on('click', $noduleGrid.removeNodule);
+            exclusiveCheckbox("[name='cectin']", "tr" /* the common parent of all checkboxes of this name*/);
```
Comment (dlwicksell) [inline docs/mockups/ctevaluation-elcap.html]: It looks like you were adjusting the indentation here, but now the indentation level is 6 spaces for the first level, and only 4 spaces for the rest. I think you missed that the previous version was indented 2 spaces at the first level, and 4 spaces for the rest.

Maybe I'm way off here, but I think you want to make it consistent, so you probably just need to get rid of the first 2 spaces for each one of these lines.

File: docs/src/_nodule-grid.jinja2 [hunk 2]
```diff
@@ -1,550 +1,669 @@
...
+                                        <option value="f"> Follow-up CT</option>
+                                        <option value="a"> Abx + follow-up CT</option>
+                                        <option value="e"> PET</option>
+                                        <option value="c"> CT w contrast</option>
+                                        <option value="b"> CT-guided biopsy</option>
+                                        <option value="p"> Bronchoscopy</option>
+                                        <option value="r"> Resection</option>
+                                    </select>
+                                </div>
+                            </td>
```
[the remark this answers is not in the document]
Comment (djdenv) [reply docs/src/_nodule-grid.jinja2]: correct, has been resolved.

File: docs/mockups/pet.html [hunk 3]
```diff
@@ -294,570 +294,5719 @@ <h2>
...
+             <label class="aria-hidden" for="cect4su">
+              <abbr title="Standard Update Value">
+               SUV
+              </abbr>
+             </label>
+             <input class="form-control numeric decimalformat" id="cect4su" name="cect4su" required="required" size="4" tabindex="404" type="text"/>
+            </div>
+           </td>
+           <td class="form-inline" data-nodule-id="5" style="display:none;">
+            <div cla
```
[the remark this answers is not in the document]
Comment (djdenv) [reply docs/mockups/pet.html]: resolved in source file

File: docs/mockups/ctevaluation-elcap.html (hunk 1 above)
Comment (djdenv) [reply docs/mockups/ctevaluation-elcap.html]: The indentation you see in these HTML files is irrelevant. These HTML files go through the Jinja templating engine and then the BeautifulSoup utility to make them "pretty" HTML 5 documents. The final format these are in is based on those utilities and has very little to do with the original source files. The particular line you noted is actually in _nodule_grid.jinja2 file. That is formatted using 4 spaces throughout and is very readable (IMHO). Though the tabbing isn't awfully important in the final HTML, I'll note that what is on each line **IS** important for the Mumps forms processor to work properly.

**Pull request merged**